index.md

```diff
@@ -12,22 +12,32 @@ postal-code:
 
 # Welcome to OWASP Bristol Chapter 
 
-The OWASP Bristol Chapter  is  run by [Katy Anton](mailto:katy.anton@owasp.org) and [Craig Francis](mailto:craig.francis@owasp.org) .
+The OWASP Bristol Chapter is run by [Katy Anton][katy], [Craig Francis][craig] and [Jon Gadsden][jon].
```

Comment:
Should the email addresses be removed?

Reply:
Oh, interesting, just noticed the footer links, not seen those before.

Reply:
yes, it is a common technique now, and makes the raw markdown much more readable. Also keeps the line lengths shorter, better all round